frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -31,26 +34,46 @@ const props = defineProps({
         type: Array as PropType<FamApplicationUserRoleAssignmentGet[]>,
         default: [],
     },
+    applicationAdmins: {
+        type: Array as PropType<FamAppAdminGet[]>,
+        default: [],
+    },
 });
 
 const userRoleAssignments = shallowRef<FamApplicationUserRoleAssignmentGet[]>(
     props.userRoleAssignments
 );
 
+const applicationAdmins = shallowRef<
+    FamAppAdminGet[]
+>(props.applicationAdmins);
+
 onUnmounted(() => {
     resetNotification();
 });
 
+const tabHeader = computed(() => {
+    return selectedApplicationShortDisplayText.value === 'FAM'
+            ? 'Application admins'
+            : 'Users';
+})
+
 const onApplicationSelected = async (e: DropdownChangeEvent) => {
     setSelectedApplication(e.value ? JSON.stringify(e.value) : null);
-    userRoleAssignments.value = await fetchUserRoleAssignments(
-        selectedApplication.value?.application_id
-    );
+    if(e.value.application_id === 2) {
```

Comment:
we need to change the logic here, cause we have more than 2 applications. If application_id is 1, then fetch admins, otherwise fetch user role assignments. And I could recommend create constant variable on top of this file for the the FAM application name and application id, and use that constant variable in the file.

Reply:
I'll do that!